trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/library/ILibraryModel.java

```diff
@@ -52,24 +99,47 @@ public record TypeModel(String typeName, ICType cType) {
 		// empty
 	}
 
+	/**
+	 * Model of a predefined constant, consisting of the name of the type and our translated model, i.e., a
+	 * {@code ICType}) and an {@code Expression}.
```

Comment:
This doesn't seem to match the record declaration.

Reply:
Thanks for the hint! I changed the signature, but forgot to update the documentation.